lib/Chat/SystemMessage/Listener.php

```diff
@@ -122,6 +122,19 @@ public static function register(IEventDispatcher $dispatcher): void {
 				'oldName' => $event->getOldValue(),
 			]);
 		});
+		$dispatcher->addListener(Room::EVENT_AFTER_DESCRIPTION_SET, static function (ModifyRoomEvent $event) {
+			$room = $event->getRoom();
+			/** @var self $listener */
+			$listener = \OC::$server->query(self::class);
```

Comment:
query is deprecated, use
```suggestion
			$listener = \OC::$server->get(self::class);
```